commands/utility/remindme.js

```diff
@@ -0,0 +1,48 @@
+const Reminder = require('../../src/models/Reminders.js')
+
+exports.run = async (handler, message, args, pre) => {
+	if (!args.days && !args.hours && !args.minutes && !args.seconds) {
+		return message.channel.send('Days, hours, minutes, or seconds argument must be given.')
+	}
+
+	// get total milliseconds
+	var ms = ((args.days ? args.days : 0) * 86400000) + ((args.hours ? args.hours : 0) * 3600000) + ((args.minutes ? args.minutes : 0) * 60000) + ((args.seconds ? args.seconds : 0) * 1000)
+
+	// if ms > 32bit integer max size
+	if (ms > 2147483646) {
```

Comment:
You should use `Number.MAX_SAFE_INTEGER` here instead of that magic number. (`Number.MAX_SAFE_INTEGER` is also much larger than that.)